md-render/HHVM/UserDocumentation/IncludeExamplesFilter.rb

```diff
@@ -51,7 +51,49 @@ def call
             File.read(full_path),
             language: 'Hack',
           );
-          node.replace(code)
+          code_node = node.replace(code)
+          # If the example has HHVM output associated with it, then print it
+          # after the code example. However, if both a .example.hhvm.out and
+          # .hhvm.expect exist, that is an error
+          output_header = doc.document.create_element(
+            'em',
+            "Output",
+          );
+          if (File.exists? (full_path + ".hhvm.expect")) ^ (File.exists? (full_path + ".example.hhvm.out"))
+            if File.exists? (full_path + ".hhvm.expect")
+              output_path = full_path + ".hhvm.expect"
+            else
+              output_path = full_path + ".example.hhvm.out"
+            end
+            output = doc.document.create_element(
+              'pre',
+              File.read(output_path),
+              language: 'Bash',
```

Comment:
Oh wait - I did do that on purpose actually.

This is output like you would see at the shell, from the command line. That's why I chose Bash.